install_pim/manual/system_requirements/system_configuration.rst.inc

```diff
@@ -72,8 +72,7 @@ First, create the file ``/etc/apache2/sites-available/akeneo-pim.local.conf``
 
     * Replace ``/path/to/installation`` by the path to the directory where you want to install the PIM.
     * Replace ``pim-community-standard`` by ``pim-enterprise-standard`` for enterprise edition.
-    * ``/run/php/php7.4-fpm.sock`` is the default socket path. If you changed it in ``/etc/php/7.4/fpm/pool.d/www.conf``, change it in the virtual host too.
-    * Don't forget to add the ``web`` directory of your Symfony application.
+    * ``/run/php/php8.0-fpm.sock`` is the default socket path. If you changed it in ``/etc/php/8.0/fpm/pool.d/www.conf``, change it in the virtual host too.
```

Comment:
What was the purpose of the following sentence? `* Don't forget to add the ``web`` directory of your Symfony application.`
Should we replace the `web` directory with `public` to avoid some potential typo during the writing of the configuration?

Reply:
I don't know. As per the issue, the web directory is not exist anymore.